virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -157,6 +154,9 @@ Removed features are those that were deprecated in earlier releases. They are no
 //CNV-45783
 * CentOS 7 and CentOS Stream 8 are now in the End of Life phase. As a consequence, the container images for these operating systems have been removed from OpenShift Virtualization and are no longer link:https://access.redhat.com/articles/4234591#community[community supported].
 
+//CNV-51829: remove tekton-tasks-operator informaiton form release notes; CNV-41244: Release note: Deprecate
+* The `tekton-tasks-operator` is removed. The Tekton tasks and example pipelines are now deployed by the `ssp-operator`.
```

Comment:
This is not correct. The tekton tasks are not deployed by ssp operator since 4.16.
They are now available via artifacthub.io.